packages/vscode-extension/README.md

```diff
@@ -1,64 +1,85 @@
+
+## <font size="6"> <span style="color:gray"> Getting Started </font> </span>
+
+### <font size="5"> <span style="color:gray"> How to Trigger the Command Palette </font>
+Interact through the Command Palette to use the SuiteCloud Extension for Visual Studio Code. The Command Palette allows you to perform several actions in a straightforward way, improving the user experience to the most. To trigger it, use the shortcut, which is Ctrl+Shift+P by default, or click **View** > **Command Palette**. 
+
+<p align="center"><a href="#"><img src="resources/CommandPalette.png" alt="Command Palette" width="500" height="600"></a></p>
 
-Read the detailed list of prerequisites in [NetSuite's Help Center](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_159223197655.html).
+### <font size="5"> <span style="color:gray"> Setting Up an Account </font> </span>
+To start using the SuiteCloud Extension for Visual Studio Code, you first need to set up the NetSuite accounts you want to customize. The account set up includes adding an account to the extension and setting an account as default. 
 
+To add an account, follow these steps:
+1. Click Ctrl+Shift+P to open the Command Palette.
+2. Enter **SuiteCloud** and from the dropdown list, select **SuiteCloud: Set Up Account**.
+3. From the dropdown list, select **New authentication ID**.
+4. From the dropdown list, select one of the following options:
+    * Browser-based authentication
+    * Save a TBA token issued in NetSuite
+5. In the text field, enter an authentication ID.
+6. If you selected the browser-based authentication, press Enter to confirm your NetSuite domain and follow the instructions in the browser prompt. If you selected to save an issued TBA token, do the following:
+    * Enter your account ID.
+    * Enter the token ID of the TBA token you previously issued in NetSuite.
+    * Enter the token secret of the TBA token you previously issued in NetSuite.
+Your account is successfully set up. </br>
 
-## SuiteCloud Extension commands
-The following commands are available through the Command Palette:
-- [Update Single Object with Account Object](https://system.netsuite.com/app/help/helpcenter.nl?fid=subsect_160147382361.html)
-- [Upload File](https://system.netsuite.com/app/help/helpcenter.nl?fid=subsect_160147414469.html)
-- [List Objects](https://system.netsuite.com/app/help/helpcenter.nl?fid=subsect_160163955513.html)
-- [Setup Account](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_160147609118.html)
-- [Deploy Project](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_160147342366.html)
-- [Add Dependency References to the Manifest](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_160147339580.html)
-- [Import Objects]
-- [Import Files]
-- [Create Project]
-- [Create Files]
+To set an account as default for a project, follow these steps:
+1. Click Ctrl+Shift+P to open the Command Palette.
```

Comment:
same. not sure if 'click' is right shortcut.

Reply:
Changed accordingly.